### PR TITLE
Fix/cdn advanced qa

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/cdn-advanced`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | yes
| Tickets          | MANAGER-7065
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
CDN advanced bugfixes for QA

### CODE REVIEW

**Review by varun257 — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

File: packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html [hunk 1]
```diff
@@ -288,11 +287,13 @@ <h4 class="oui-box__heading" data-translate="hosting_cdn_shared_option_prewarm_t
                 <div class="mt-3">
                     <p class="mb-0"><strong data-translate="hosting_cdn_shared_option_prewarm_quota"></strong></p>
                     <p data-ng-bind="$ctrl.getPrewarmQuotaUsage()"></p>
-                    <progress
-                        class="w-25 oui-progress oui-progress_info"
-                        value="prewarm.extra.usage"
-                        max="prewarm.extra.quota">
-                    </progress>
+                    <div class="oui-input-group oui-input-group_l">
+                        <progress
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html:291]: **thought:** Not sure that wrapping `progress` element inside a `oui-input-group` make sense.

**question:** Couldn't be replaced by [`<oui-progress>`](https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-progress-webcomponent--default)?

File: packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html [hunk 2]
```diff
@@ -103,7 +103,7 @@ <h4 class="oui-box__heading" data-translate="hosting_cdn_shared_option_http_geol
              data-ng-init="prefetch = $ctrl.getOption($ctrl.model.optionTypes.PERF, $ctrl.cdnOptionTypeEnum.PREFETCH).api">
 
             <h4 class="oui-box__heading" data-translate="hosting_cdn_shared_option_prefetch_title"></h4>
-            <p data-translate="hosting_cdn_shared_option_prefetch_info"></p>
+            <span data-translate="hosting_cdn_shared_option_prefetch_info"></span>
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html:106]: **question:** Why not prefer wrapping the `<oui-switch>` element into a `<oui-field>`?
see: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-switch-webcomponent--validation

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html:291]: Yep, i completed with my current fixes.

File: packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html]: Oui-field had a large bottom padding which we can't managed.

**Pull request merged**